### PR TITLE
fix reference for definition of end-of-candidates" Issue841

### LINKED ISSUE #841: Wrong refs for end-of-candidates SDP
Agree, the references in Sections 5.2.2 and 5.3.2 should be updated accordingly. 

> On Wed, Oct 11, 2017 at 12:23 AM, Christer Holmberg <christer.holmberg@ericsson.com> wrote:
> Hi,
> 
> JSEP references draft-ietf-ice-trickle for the SDP ‘end-of-candidates’ attribute.
> 
> But, draft-ice-trickle only talks about a generic end-of-candidates indication. For SDP, the actual SDP attribute is defined in draft-ietf-mmusic-trickle-ice-sip.
>  Shouldn’t JSEP reference that spec instead?
>

### PR DESCRIPTION
**Pull request by fluffy**
Fixes #841

### CODE REVIEW

**Comment by fluffy** [comment]
This is the only instance that seemed wrong. The others were already fixed.

**Comment by ajeanmahoney** [comment]
LGTM

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -1765,7 +1765,7 @@ Original:
           the rules defined for Trickle ICE. Otherwise, the
           IceCandidate is treated as an end-of-candidates indication,
           as defined in
-          <xref target="RFC8838" format="default"/>.</t>
+          <xref target="RFC8840" sectionFormat="comma" section="8.1"/>.</t>
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml:1768]: I think we want RFC8840, Section 14 here, as the text isn't referring to a=end-of-candidates specifically.

**Review by juberti — CHANGES_REQUESTED** [review]

File: draft-ietf-rtcweb-jsep.xml (hunk 1 above)
Comment (ajeanmahoney) [reply draft-ietf-rtcweb-jsep.xml:1768]: RFC8840, Section 14, is the References section. 
Do you mean RFC 8838, Section 14 ("Receiving an End-of-Candidates Indication")?

Comment (juberti) [reply draft-ietf-rtcweb-jsep.xml:1768]: oops, I had meant RFC 8838, as before.

Comment (fluffy) [reply draft-ietf-rtcweb-jsep.xml:1768]: Makes sense. I will update the PR.

**Comment by fluffy** [comment]
This updates the PR per Justin's suggestion

**Review by juberti — APPROVED** [review]

**Pull request merged**